src/passes/MinifyImportsAndExports.cpp

```diff
@@ -169,7 +177,7 @@ struct MinifyImportsAndExports : public Pass {
     }
     module->updateMaps();
     // Emit the mapping.
-    for (auto& pair : oldToNew) {
+    for (auto& pair : newToOld) {
```

Comment:
Why are you now outputting the mapping in reverse?

Reply:
The code was actually wrong before - it was called `oldToNew` but had the opposite data ;) This PR also corrects that.